is08/activation.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2018 British Broadcasting Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from GenericTest import NMOSTestException
+from is08.calls import Call
+from is08.testConfig import globalConfig
+import re
+
+
+class Activation:
+
+    IMMEDIATE_ACTIVATION = 'activate_immediate'
```

Comment:
It would be nice if these were available to the IS-05 tests too. It may be too late to make this all fit the existing pattern of NMOSUtils subclassed by IS08Utils etc, but if they could at least be provided in NMOSUtils (or just defined in that file) and imported here it would help.

Reply:
I don't know how close IS-08 has ended up to 05 in the end in terms of activations, but if it's close enough and there's the potential to refactor the 05 tests to use this class in the future it would be nice to make it as easy as possible. I don't know if there are any actions which can be taken now to ease this.

Reply:
It's all a bit complicated. While the activation object looks the same IS-08 Activations come back with an ID, where as IS-05 ones don't. IS-08 activations also can be deleted (and there can be many in existence at once), so while the JSON looks familiar they behave a bit differently.

I've factored out those constants into NMOS Utils for now anyway, as that does seem sensible. Doing more than that may prove futile because of these differences however.